### PR TITLE
Maps the areas in the main hallway's on delta map

### PR DESCRIPTION
**Pull request by Fluffyzack21**
## What Does This PR Do
 Maps mainly all the main hallways and at arrivals, and separating the areas apart so there not one big area, so not pushing stress on air alarms, and APC's, and it makes it very organized for the next person to work on

## Why It's Good For The Game
 No longer puts stress on things like air alarms when sucking air, and each part of a main hallway of the station now has there own area to prevent this, and has not been done since delta was first ever made and other parts as well

## Images of changes
 
image
Screenshot 2024-02-12 094301
Screenshot 2024-02-12 095327
Screenshot 2024-02-12 101229
image
Screenshot 2024-02-12 110224

## Testing
 Compiled, checked if the hallways were getting power, and named correctly.

## Changelog
:cl:
Tweak: Added cargobay lobby on delta station
Tweak: Changed the areas around central primary, port hallway, starboard hallway and for primary hallway, arrivals, aft hallway to there correct areas on delta station
Add: Put some air alarm, fire alarms, and APC's and added some vents that were needed central primary hallway around Delta station
:cl:

### CODE REVIEW

**Comment by Henri215** [comment]
image
Fix the typo and specify what station you talking about

**Review by DGamerL — CHANGES_REQUESTED** [review]
image

Fix these please

File: _maps/map_files/Delta/delta.dmm [hunk 1]
```diff
@@ -70720,6 +70735,21 @@
 	},
 /turf/simulated/floor/plasteel/white,
 /area/station/medical/medbay)
+"gfR" = (
+/obj/structure/cable{
+	d2 = 8;
+	icon_state = "0-4"
+	},
+/obj/machinery/power/apc{
+	dir = 8;
```
Comment (Henri215) [inline _maps/map_files/Delta/delta.dmm:70744]: Same thing Warriorstar said over your meta PR, no need for the dir varedit.

Comment (Fluffyzack21) [reply _maps/map_files/Delta/delta.dmm:70744]: fixed

**Review by Henri215 — APPROVED** [review]
Lewcs Good To Me

**Review by Henri215 — CHANGES_REQUESTED** [review]
1.  Same things as your meta hallways PR, remove the light switches and properly fill the "What Does This PR Do"

2. You removed the scrubber but forgot to remove the manifold here.

image


3. Any reason to move this status display to the other wall? 

image

4. One extra cable here.

image

5. Accidental cable placed on that wall

image


6. A second camera is really necessary here?

image

**Review by Henri215 — APPROVED** [review]
Looks good to me now.

image

Just remove the mention of light switches on your changelog.

**Review by S34NW — APPROVED** [review]

**Pull request merged**